static/app/views/performance/trends/chart.tsx

```diff
@@ -96,8 +96,8 @@ function getIntervalLine(
     return [];
   }
 
-  const seriesStart = parseInt(series[0].data[0].name as string, 0);
-  const seriesEnd = parseInt(series[0].data.slice(-1)[0].name as string, 0);
+  const seriesStart = parseInt(series[0].data[0].name as string, 10);
+  const seriesEnd = parseInt(series[0].data.slice(-1)[0].name as string, 10);
```

Comment:
@k-fish could you confirm that this was your intention? The updated eslint rules sees using `0` as the `radix` parameter as an error.

Though the behavior of `0` IS defined for `parseInt` here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt#description

> A value passed as the radix argument is coerced to a Number (if necessary), then if the value is 0, NaN or Infinity (undefined is coerced to NaN), JavaScript assumes the following:
> 
> 1. If the input string begins with "0x" or "0X" (a zero, followed by lowercase or uppercase X), radix is assumed to be 16 and the rest of the string is parsed as a hexadecimal number.
> 2. If the input string begins with any other value, the radix is 10 (decimal).

Reply:
Not sure if you actually... wanted some kind of parsing on hexadecimal values?

Reply:
lol, no, was just a typo, glad the new eslint caught it.